examples/ephemeral/main.tf

```
@@ -62,8 +62,9 @@ module "runners" {
   # # Example of simple pool usages
   # pool_runner_owner = "philips-test-runners"
   # pool_config = [{
-  #   size                = 3
-  #   schedule_expression = "cron(* * * * ? *)"
+  #   size                         = 3
+  #   schedule_expression          = "cron(* * * * ? *)"
+  #   schedule_expression_timezone = "Australia/Sydney"
```

Maybe also update the exmple cron expression that make sense for a certain time zone.
```suggestion
  #   size                         = 3
  #   schedule_expression = "cron(0/3 14 * * ? *)" # every 3 minutes between 14:00 and 15:00
  #   schedule_expression_timezone = "Europe/Amsterdam"

```